pitch.md

```diff
@@ -1,44 +1,90 @@
-# Project Title (replace me)
+# BarScript
 
 ## Overview
 _What's this project about, briefly?_
 
+This project is about creating a language to specify coffee/alcoholic drinks, so they can be made to your taste by an automated machine. We already have some automated coffee machines, but these machines usually have a limited number of adjustable settings and are considered by many to be inferior to an actual barista's work for this reason. Theoretically, if a project like this was wildly successful it'd eliminate the need for bartenders/baristas.
```

Comment:
I may do that. My hope is that the language is easily extendable to the other kind of drink too.